WoWDatabaseEditor.Common/WDE.Common/Database/CharacterRaces.cs

```diff
@@ -44,5 +46,6 @@ public enum CharacterRaces : uint
     AllWoD = AllMoP,
     AllLegion = AllWoD | Nightborne | HighmountainTauren | VoidElf | LightforgedDraenei,
     AllBfA = AllLegion | ZandalariTroll | KulTiran | DarkIronDwarf | Vulpera | MagharOrc | Mechagnome,
-    AllShadowlands = AllBfA
```

Comment:
Could you also add new faces to AllAlliance/AllHorde enum?

Reply:
> Could you also add new faces to AllAlliance/AllHorde enum?

Talk to me on Discord as I have a question regarding this matter. I have given you my Discord ID in a comment below.